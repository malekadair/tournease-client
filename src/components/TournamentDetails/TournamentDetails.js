import React, { Component } from "react";

class TournamentDetails extends Component {
  render() {
    const {
      title,
      date,
      time,
      game,
      fee,
      address,
      moreDetails,
      id
    } = this.props.tournament;
    return (
      <div>
        <h1>Tournament Name: {title}</h1>
        <h2>Date: {date}</h2>
        <h2>Start Time: {time}</h2>
        <h2>Game Type: {game}</h2>
        <h2>Entry Fee: ${fee}</h2>
        <h2>Address of Venue: {address}</h2>
        <p>Things to note: {moreDetails}</p>
      </div>
    );
  }
}

export default TournamentDetails;
