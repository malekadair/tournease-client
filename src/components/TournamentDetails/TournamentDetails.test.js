import React from "react";
import toJson from "enzyme-to-json";
import TournamentDetails from "./TournamentDetails";

describe(`TournamentDetails component`, () => {
  const props = {
    id: "a",
    name: "test-class-name"
  };

  it("renders a .TournamentDetails by default", () => {
    const wrapper = <TournamentDetails />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("renders the TournamentDetails given props", () => {
    const wrapper = <TournamentDetails {...props} />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
