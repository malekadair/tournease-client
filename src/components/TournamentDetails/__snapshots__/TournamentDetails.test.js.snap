// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TournamentDetails component renders a .TournamentDetails by default 1`] = `undefined`;

exports[`TournamentDetails component renders the TournamentDetails given props 1`] = `undefined`;
